### PR TITLE
feat:adding signup page to E-commerce

### PR DESCRIPTION
**Pull request by mariyasada**
add signup page to E-commerce
adding signup page link to login page
adding padding-right to sidebar list item

### CODE REVIEW

File: stylesheets/signup/signup.css [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+{
+    min-height: 80vh;
+    max-width:80%;
+    display:flex;
+    flex-direction: column;
+    box-shadow: 2px 2px 5px var(--dark-stone);
+    align-items: center;
+    margin: auto;
+}
+.text-size-xl
```
Comment (kotalwar-krishna) [inline stylesheets/signup/signup.css:12]: one suggestion after your completion or before completion make typography seprated.  so you can reuse in other pages also

**Review by kotalwar-krishna — APPROVED** [review]
Good to merge.
The preview of sign-up page looks nice.

**Review by t007rushi — CHANGES_REQUESTED** [review]
add cursor: pointer for cards, There is no need to fix footer, You can fix header just suggestion

File: stylesheets/signup/signup.css (hunk 1 above)
Comment (mariyasada) [reply stylesheets/signup/signup.css:12]: i am already reusing the text-utilities classes from CL

**Review by t007rushi — APPROVED** [review]

**Pull request merged**